Simplenote/Information/NoteInformationViewController.swift

```diff
@@ -177,6 +179,12 @@ private extension NoteInformationViewController {
 
         additionalSafeAreaInsets = UIEdgeInsets(top: headerStackView.frame.maxY, left: 0, bottom: 0, right: 0)
     }
+    
+    func configureDragBar() {
+        if navigationController != nil {
+            dragBar.isHidden = true
```

Comment:
Nipticky (cosmetic, feel free to disregard!!). You can simplify this one a bit:

```
dragBar.isHidden = navigationController != nil
```

Reply:
I like it, much cleaner